example/test.spec.js

```diff
@@ -40,6 +40,7 @@ describe('describe', function() {
 
   beforeEach(function() {
     foo = 'bar';
+    return setTimeout(() => {}, 100);
```

Comment:
Is it necessary for user?
Could you clarify it please?

Reply:
It's needed just for making test more slowly to avoid mixes tests and hook in case of the same start time.

Reply:
Ok. Got it